app/views/pages/about.html.erb

```diff
@@ -8,4 +8,4 @@
   to teach web development with
```

Comment:
Tìm hiểu I18n và sử dụng vào toàn bộ file html.erb nhé
tự soft check và sửa nhé em